app/models/form/lettings/questions/age.rb

```diff
@@ -9,7 +9,8 @@ def initialize(id, hsh, page, person_index:)
     @inferred_check_answers_value = [{ "condition" => { "age#{person_index}_known" => 1 }, "value" => "Not known" }]
     @check_answers_card_number = person_index
     @max = 120
-    @min = 0
+    @min = 1
```

Comment:
Can a child not be 0-1?

Reply:
this is specified in ticket. Your reasonable question is somewhat handled in the specified hint text for children.